cmd/crl-updater/main.go

```diff
@@ -143,10 +150,13 @@ func main() {
 	if c.CRLUpdater.LookbackPeriod.Duration == 0 {
 		c.CRLUpdater.LookbackPeriod.Duration = 24 * time.Hour
 	}
+	if c.CRLUpdater.UpdateTimeout.Duration == 0 {
+		c.CRLUpdater.UpdateTimeout.Duration = 1 * time.Hour
+	}
 
 	saConn, err := bgrpc.ClientSetup(c.CRLUpdater.SAService, tlsConfig, scope, clk)
 	cmd.FailOnError(err, "Failed to load credentials and create gRPC connection to SA")
-	sac := sapb.NewStorageAuthorityReadOnlyClient(saConn)
+	sac := sapb.NewStorageAuthorityClient(saConn)
```

Comment:
Have SRE verify that the DSN has the proper grants.

Reply:
This isn't a DSN, its a gRPC connection to an SA instance, but regardless, SRE has made the appropriate updates.